guide/token-setup-and-validation.md

```diff
@@ -0,0 +1,351 @@
+# Token setup and validation
+
+For an overview of the different token formats, see: [Link to token formats]
+
+This guide will be divided into 2 parts: token generation and token validation.
+
+In OpenID Connect there are three types of tokens: access tokens, id tokens, and refresh tokens. (link). When this guide refers to _tokens_ it is referring to access tokens.
```

Comment:
https://openid.net/specs/openid-connect-core-1_0.html#Introduction

Reply:
> When this guide refers to _tokens_ it is referring to access tokens.

Maybe we should avoid that and be explicit by using `access token` instead of just `token` everywhere?